### ISSUE TITLE
Some questions about AL process part different between code implementation and paper description

### ISSUE THREAD

**Issue by xnnxax527** [yanxp/SSM-Pytorch#4]
Hi yanxp,

Recently I am reading the paper **Towards Human-Machine Cooperation: Self-supervised Sample Mining for Object Detection** and the code implementation.

I have some questions about the AL process part which seems a little different between the code and the paper description.

![image](https://user-images.githubusercontent.com/40298019/48117882-88f46780-e2a5-11e8-9caf-1c43e34731e5.png)

In my understanding, the framework above in the paper shows the image whether require user annotating (AL  process) depends on the prediction consistency score from image cross validation. So it means do image cross validation firstly...

![image](https://user-images.githubusercontent.com/40298019/48118272-c86f8380-e2a6-11e8-8c8f-98206d757171.png)

**But in the code, it seems a little different....**

In my understanding, the AL part just depends on the function judge_uv (cls loss), but not related to image cross validation.....

So image cross validation just works for SS part?

Maybe I misunderstood... could you help me please, thanks!

**Comment by yanxp** [comment]
hello, yes, in our experiments only the predicted  high score (SS path ) will be into image cross validation, the AL Part will not be into image cross validation.  Maybe the figure lead to misread. Thank you for pointing out, we will update it.

**Comment by kezewang** [comment]
Dear xnnxax527,

Thanks a lot for your kind attention to our work!
Currently, this implementation assumes that the rest unlabeled images after many training iterations have been ignored by the image cross validation.  Therefore, we can directly perform AL on these images. For better understanding, we will update our code and solve this issue ASAP. 

Best Regards
